### PR TITLE
added in the necessary info for the april 18th date, and the new sponsor

### PR DESCRIPTION
**Pull request by nickgartmann**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -159,9 +160,10 @@ <h2 id="faq">FAQ</h2>
     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
+/*
```
Comment (mitchellhenke) [inline index.html]: ?

Comment (nickgartmann) [inline index.html]: Oops! Thank you.

**Comment by mitchellhenke** [comment]
:+1:

**Comment by alexnsolo** [comment]
:+1:

**Pull request merged**